docs/2.7.0/docs/daml-finance/tutorials/instrument-extension-packages/callable-bond.rst

```diff
@@ -0,0 +1,159 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Election-based lifecycling (using a callable bond)
+##################################################
+
+This tutorial describes how to define and process Elections. It builds on the previous
+:doc:`Time-based Lifecycling <fixed-rate-bond>` tutorial, which uses a fixed rate bond where all
+coupons are pre-defined and happen as time passes. In contrast, the coupons of a callable bond
+depend on whether the issuer has called the bond. Hence, a simple time event is not sufficient to
+define the state of the instrument. Instead, the lifecycling framework requires an active Election
+on each date when it is possible to call the bond. This Election is the main topic of this tutorial.
+Check out the :ref:`Lifecycling concepts <time-vs-election-lifecycling>` for more details on time
+based vs election based evolution of instruments.
+
+In this tutorial, we are going to:
+
+#. create a callable bond instrument and book a holding on it
+#. reuse the lifecycle rule and settlement factory from the fixed rate bond tutorial
+#. create the election not to call the bond
+#. process the election event to produce the effects of a coupon payment
+#. instruct settlement by presenting a bond holding
+#. settle the resulting batch atomically
+
+This example builds on the previous :doc:`Time-based Lifecycling <fixed-rate-bond>` tutorial script
```

Comment:
You already mention that this tutorial builds on the fixed rate bond tutorial, so I would remove this paragraph